src/pages/_document.tsx

```diff
@@ -12,6 +13,7 @@ class LensterDocument extends Document {
         <body>
           <Main />
           <NextScript />
+          <HexagonClipConfig />
```

Comment:
**suggestion:** I'm worried it will be rendered on all pages that even don't have user's avatar 😕

Reply:
thanks for suggestion
left general comment

this case would be infrequent if when the user is not logged in, and also, it is like other configs

I have suggested 2 workarounds, with extra job